devicehive-common/src/main/java/com/devicehive/model/rpc/CommandSubscribeRequest.java

```diff
@@ -93,11 +107,6 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return "CommandSubscribeRequest{" +
-                "subscriptionId='" + subscriptionId + '\'' +
-                ", device='" + device + '\'' +
-                ", names=" + names +
-                ", timestamp=" + timestamp +
-                '}';
+        return "CommandSubscribeRequest{" + "subscriptionId=" + subscriptionId + ", device=" + device + ", names=" + names + ", timestamp=" + timestamp + ", limit=" + limit + '}';
```

Comment:
Let's avoid long code lines. Maybe it's better to use `java.lang.String.format` method? Also, please double-check that single quotes are not missed.